src/WebJobs.Script/Binding/Http/JsonResolver.cs

```diff
@@ -0,0 +1,29 @@
+using System.Net;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using Newtonsoft.Json.Serialization;
+using System.Reflection;
+using System.Collections.Generic;
+
+namespace Microsoft.Azure.WebJobs.Script.Binding
+{
+    public class IgnoreErrorPropertiesResolver : DefaultContractResolver
+    {
+        protected override JsonProperty CreateProperty(MemberInfo member, MemberSerialization memberSerialization)
+        {
+            JsonProperty property = base.CreateProperty(member, memberSerialization);
+
+            List<string> ignoreProperties = new List<string>()
```

Comment:
Rather than new up a list each time, create this in a ctor. Also, you can probably reuse an instance of this type rather than newing one up each time?